.github/workflows/workflow.yml

```diff
@@ -53,9 +55,9 @@ jobs:
         API_TOKEN_GITHUB: ${{ secrets.PUBLISH_BUILD_GITHUB_TOKEN }}
       with:
         source-directory: './actions/submit-signing-request/dist'
-        destination-github-username: 'volbobvol'
-        destination-repository-name: 'action-target'
+        destination-github-username: ${{ env.TARGET_REPO_OWNER }}
+        destination-repository-name: ${{ env.SUBMIT_SIGNING_REQUEST_TARGET_REPO }}
         user-name: GHA build process
         create-target-branch-if-needed: true
-        commit-message: ${{ github.event.head_commit.message }}
-        target-branch: ${{ github.ref_name }}
\ No newline at end of file
+        commit-message: Original commit message - ${{ github.event.head_commit.message }}
```

Comment:
Suggestion for the commit message: Built from "<source-repo> with build <build-id> (commit <sha1-hash>)"